packages/lib/src/components/Dropin/elements/createInstantPaymentElements.ts

```diff
@@ -9,6 +9,6 @@ import UIElement from '../../UIElement';
  * @param create - Reference to the main instance `Core#create` method
  */
 const createInstantPaymentElements = (paymentMethods: PaymentMethod[] = [], props, create): Promise<UIElement[]> | [] =>
-    paymentMethods.length ? createElements(paymentMethods, { ...props, isInstantPayment: true, showPayButton: true }, create) : [];
```

Comment:
We need the `showPayButton: true` here for Instant payment methods - they are not supposed to be used with custom pay button

Reply:
ok